askama_derive/src/config.rs

```diff
@@ -131,41 +131,18 @@ impl<'a> Config<'a> {
     }
 }
 
-#[derive(Debug)]
-pub(crate) struct Syntax<'a> {
-    pub(crate) block_start: &'a str,
-    pub(crate) block_end: &'a str,
-    pub(crate) expr_start: &'a str,
-    pub(crate) expr_end: &'a str,
-    pub(crate) comment_start: &'a str,
-    pub(crate) comment_end: &'a str,
-}
-
-impl Default for Syntax<'static> {
-    fn default() -> Self {
-        Self {
-            block_start: "{%",
-            block_end: "%}",
-            expr_start: "{{",
-            expr_end: "}}",
-            comment_start: "{#",
-            comment_end: "#}",
-        }
-    }
-}
-
-impl<'a> TryFrom<RawSyntax<'a>> for Syntax<'a> {
+impl<'a> TryInto<Syntax<'a>> for RawSyntax<'a> {
```

Comment:
Coherence. At the end of this PR, `Syntax` no longer lives in askama_derive, so we're not allowed to implement the foreign trait `TryFrom` for the foreign type `Syntax`. However, we can implement `TryInto` for `RawSyntax`, since that's local. This is explained in the docs for [`Into`](https://doc.rust-lang.org/stable/std/convert/trait.Into.html), which the `TryInto` docs link to.